### ISSUE TITLE
[BUG] Mac Beta: Screen is blank when starting the application, unable to do anything.

### ISSUE THREAD

**Issue by ahalbert** [getdevspace/open-management#11]
OS X 10.15.4

I installed DevSpace from here https://github.com/getdevspace/open-management/releases/tag/v1.0.0-beta1 on mac os. When I launched the application, the screen was blank.

Opening up dev tools indicates that the html of the page is an empty <body> tag. 

Attached is a screenshot of what I'm seeing.

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/849804/82091501-768e9180-96c5-11ea-96ac-2ad426817e85.png">

**Comment by gabrielgatu** [comment]
Hey @ahalbert ! Thanks again for the report!
Do you mind If I ask you some more questions about the error?

1) Could you tell me which version of MacOS do you have?
2) Did you open the .pkg with right click -> open or did you just double click on it?
3) Did you try to open the app with right click?
4) Did it show any dialogs after clicking on it?
5) Do you happen to know if you have MacOS Gatekeeper disabled or enabled?  (https://support.apple.com/en-us/HT202491)

I'm sorry for so many questions, but I'm trying to narrow it down to be able to replicate it :) The main cause I think it has to be related to the internal web server not being able to start (on port 19955).

**Comment by ahalbert** [comment]
Thank you for your response.

1. OS X 10.15.4
2. I opened the pkg with a right click. It appeared to install devspace sucessfully
3. Opening with right/left did not change anything. 
4. I think there were dialogs the first time I opened it, but not sure. I don't see any now.
5. Gatekeeper is disabled. 

I can also check  port 19955.

**Comment by gabrielgatu** [comment]
Thanks! That's really interesting! I've just tried with an external macbook I've used before to test the app (and not the one used to sign the app) and everything is working as normal. The OS version is also the same.

I'm gonna try to ask a friend to install the app as well and let me know, maybe there's some configuration I did on my macs that allow the app to run correctly.

**Comment by ahalbert** [comment]
`netstat -vanp tcp`

Told me that nothing is runng on 19955

**Comment by gabrielgatu** [comment]
Ok, good and bad news here: I've managed to replicate the error on a friend's mac by launching directly the server exec located at Contents/Resources/app/bin/devspace-server and it's failing because of an "image not found - openssl@1.1".

Since the server is written and compiled in Rust, I suppose that during the exporting phase, it linked against this version of openssl (because of a dependency?) and it's working just fine on the dev machine since it has it.

The included openssl version on the lastest macs is 2.8.3, while the executable is searching for 1.1, I'm gonna leave this info here as well in case someone knows something more and can help me out :)

**Comment by gabrielgatu** [comment]
A possible raw solution just came to my mind: probably this library is installed by rustup while installing rust, or, less probably, it's installed by XCode.

Obviously, this is not a viable solution for the release, since I cannot ask everyone to install external tools on their mac just to run DevSpace :yum: 

In any case, could I ask you @ahalbert if you have XCode installed?

**Comment by ahalbert** [comment]
Gabriel,

I do have xcode and most developer tools installed. I can install rust packages to try and run Devspace

**Comment by gabrielgatu** [comment]
Issue resolved. Indeed it was caused by the openssl@1.1 version, which is the current stable (LTS) one, hence why a rust library I'm using is using it as a dependency.
Luckily for me the official rust docs for openssl are pretty awesome https://docs.rs/openssl/0.10.29/openssl/ and allowed me to embed the required version in the app bundle.

I've updated the pkg in the releases page, have a try with this one: https://github.com/getdevspace/open-management/releases/download/v1.0.0-beta1/DevSpace-beta-MacOS.pkg
